README.rst

```diff
@@ -64,6 +64,19 @@ variables as necessary. Once done, source the file to get all of the variables i
 
     open vnc://localhost:35900
 
+
+Creating a test course on stage
+-------------------------------
+
+By default, the tests in this repo run against our staging server. If you plan to keep this as the target environment, you must create your own course on stage (this requires an account with staff access). Using your own course will prevent collisions with other developers and ensure that your test results are accurate.
+
+To create the course:
+
+1. Log into studio.edx.org with staff credentials
+2. Create a new course, giving it a unique combination of Organization/ Course Number/ Course Run. The Course Name won't matter here, as it will be overwritten to "E2E Test Course" by the next step.
```

Comment:
Rather than saying 'a unique combination', perhaps just say use your github username as the org. That way we don't have muliple people using 'foobar' or 'test'. What do you think?

Reply:
Good point. I think I'll say that for course number instead though. it seems that you must use an existing org.